docs/cli_reference.md

```diff
@@ -363,12 +386,12 @@ Now let’s start Llama Stack Distribution Server.
 You need the YAML configuration file which was written out at the end by the `llama stack build` step.
 
 ```
-llama stack run local --name 8b-instruct --port 5000
+llama stack run --run-config ~/.llama/builds/local/conda/8b-instruct.yaml --port 5000
```

Comment:
nit: keep the argument name as `--config` across all commands (build, configure, run) no need for `--config-file` and `--run-config`